vercel/resource_deployment.go

```diff
@@ -89,6 +89,34 @@ Once the build step has completed successfully, a new, immutable deployment will
 					mapItemsMinCount(1),
 				},
 			},
+			"git_source": {
+				Description:   "A map with the Git repo information",
+				Optional:      true,
+				PlanModifiers: tfsdk.AttributePlanModifiers{tfsdk.RequiresReplace()},
+				Attributes: tfsdk.SingleNestedAttributes(map[string]tfsdk.Attribute{
+					"repo_id": {
+						Required:      true,
+						PlanModifiers: tfsdk.AttributePlanModifiers{tfsdk.RequiresReplace()},
+						Type:          types.StringType,
+						Description:   "Frontend git repo ID",
+					},
+					"ref": {
+						Required:      true,
+						PlanModifiers: tfsdk.AttributePlanModifiers{tfsdk.RequiresReplace()},
+						Type:          types.StringType,
+						Description:   "Branch or commit hash",
+					},
+					"type": {
+						Required:      true,
+						PlanModifiers: tfsdk.AttributePlanModifiers{tfsdk.RequiresReplace()},
+						Type:          types.StringType,
+						Description:   "Type of git repo, supported values are: github",
```

Comment:
Is this a restriction of the deplyoment API, or is github the only supported value?

Reply:
I thought `github` was the only supported value but it looks like they support more in the API specification.
updating

Reply:
changed